### ISSUE TITLE
Error when completing with coc-jedi

### ISSUE THREAD

**Issue by Rahlir** [neoclide/coc.nvim#3295]
## Result from CocInfo

```
## versions

vim version: NVIM v0.4.3
node version: v16.6.2
coc.nvim version: 0.0.80-a91848b38f
coc.nvim directory: /home/uhlir/.vim/plugged/coc.nvim
term: tmux-256color
platform: linux

## Log of coc.nvim

2021-08-18T15:11:57.120 DEBUG (pid:1081880) [workspace] - buffer created 1
2021-08-18T15:11:57.139 INFO (pid:1081880) [services] - registered service "languageserver.ccls"
2021-08-18T15:11:57.687 INFO (pid:1081880) [services] - registered service "jedi"
2021-08-18T15:11:57.687 DEBUG (pid:1081880) [services] - starting service: jedi
2021-08-18T15:11:57.688 INFO (pid:1081880) [services] - jedi-language-server state change: stopped => starting
2021-08-18T15:11:57.690 INFO (pid:1081880) [plugin] - coc.nvim 0.0.80-a91848b38f initialized with node: v16.6.2 after 605ms
2021-08-18T15:11:57.694 INFO (pid:1081880) [language-client-index] - Language server "jedi" started with 1081909
2021-08-18T15:11:58.181 INFO (pid:1081880) [services] - jedi-language-server state change: starting => running
2021-08-18T15:11:58.182 DEBUG (pid:1081880) [languages] - created service source jedi
2021-08-18T15:11:58.184 INFO (pid:1081880) [services] - service jedi started
2021-08-18T15:12:04.756 ERROR (pid:1081880) [node-client] - request error on "nvim_call_function" [
  'coc#float#create_pum_float',
  [
    0,
    0,
    [
      'cred_unhashed = "{:s}:{:s}".format(SERVER_PARAMS["username"], SERVER_PARAMS["password"])',
      '—'
    ],
    {
      codes: [Array],
      highlights: [],
      maxWidth: 80,
      pumbounding: [Object]
    }
  ]
] Vim(syntax):E484: Can't open file syntax/text.vim Error
    at kN.request (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:30:34613)
    at kN.call (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tx.show (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:25901)
    at WY.onPumChange (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:36822)
2021-08-18T15:12:04.757 ERROR (pid:1081880) [server] - unhandledRejection  Promise {
  <rejected> Error
      at kN.request (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:30:34613)
      at kN.call (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:33:3671)
      at tx.show (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:25901)
      at WY.onPumChange (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:36822)
} Error
    at kN.request (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:30:34613)
    at kN.call (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tx.show (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:25901)
    at WY.onPumChange (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:36822)
```
## Describe the bug

When I start editing a python file (for python, I am using **coc-jedi**, so the error is probably related to that extension) and I trigger autocompletion as I type a name of a variable defined above, autocompletion menu pops up and when I select the desired item, Vim throws an error:
```
[coc.nvim]: UnhandledRejection: request error nvim_call_function - Vim(syntax):E484: Can't open file syntax/text.vim
Error
    at kN.request (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:30:34613)
    at kN.call (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tx.show (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:25901)
    at WY.onPumChange (/home/uhlir/.vim/plugged/coc.nvim/build/index.js:238:36822)
```
Clearly, there is a problem with creating the preview window which is supposed to show the code snippet where
the variable is defined. I guess the problem is that vim is trying to highlight that code using file syntax/text.vim instead of syntax/python.vim. Note, however, that the preview window actually is created without any problems. Only after the error is thrown the preview window is not closed correctly. In fact, there is a workaround for this bug which is to create an empty text.vim with `touch ~/.vim/syntax/text.vim`. With this workaround the preview window showing the code snippet (and possible a corresponding pydoc) is, however, not highlighted as python code - clearly, vim highlights the content of the preview window using the empty text.vim file.

This error seems to happen only on autocompletion. When I use `CocActionAsync('doHover')`, everything works just fine.

Moreover, reverting back to the original tagged release of coc.nvim `v0.0.80` fixes the issue. So, clearly, the problem is with some newer commit.

I know that this could be an issue with `coc-jedi`, so far I have not encountered the issue with any other completion source. However, since the bug is coming from coc.nvim and I couldn't really figure out what could `coc-jedi` be doing wrong, I opened this issue here. If it's indeed an issue with `coc-jedi`, I will open an issue there.

## Reproduce the bug

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  inoremap <silent><expr> <c-space> coc#refresh()
  ```

- Start (neo)vim with command: `vim -u mini.vim test.py`

- Write
```python
spam = "hello world"
sp
```
- Press `<c-space>` and select the item spam from the completion menu

**Comment by chemzqm** [comment]
Don't know how could it be `text`, you can use `let g:coc_markdown_disabled_languages = ['text']` to disable it.

**Comment by Rahlir** [comment]
Well I have looked at node communication with nvim and found out that node is calling the function `coc#float#create_pum_float` with arguments (for a sample completion item):
```16:28:23.447 DEBUG [transport] - request to nvim: 11,nvim_call_function,[
  "coc#float#create_pum_float",
  [
    0,
    0,
    [
      "SERVER_PARAMS = { \"username\": \"admin\", \"password\": \"3dA9gmtxapJE9Fe\", \"host\": \"127.0.0.1\", \"port\": \"5984\" }",
      "—"
    ],
    {
      "codes": [
        {
          "filetype": "python",
          "startLine": 0,
          "endLine": 1
        },
        {
          "filetype": "text",
          "startLine": 2,
          "endLine": 2
        }
      ],
      "highlights": [],
      "maxWidth": 80,
      "pumbounding": {
        "col": 6,
        "row": 10,
        "height": 1,
        "width": 21,
        "scrollbar": false
      }
    }
  ]
]
```
Clearly, the argument `filetype: "text"` is wrong. It should be "txt" or "python". So there is a bug somewhere. I am not sure this should be closed and ignored just by disabling "text" highlighting. Moreover, pydoc in preview windows is not highlighted. So, somewhere, the function is called with "filetype: txt" instead of python.

**Comment by chemzqm** [comment]
It's fixed by 6b77441, the error would be ignored.
